.vscode/tasks.json

```diff
@@ -0,0 +1,24 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "build",
+            "type": "shell",
+            "command": "./build.sh -d",
```

Comment:
Is this -d debug? Could the label be a bit clearer?

Reply:
This is a regular flag that just indicates to the CMAKE to set debug flags for line coverage etc.